projects/Mallard/src/components/chevron.tsx

```diff
@@ -2,9 +2,15 @@ import React from 'react'
 import { color as themeColor } from '../theme/color'
 
 import Svg, { G, Path } from 'react-native-svg'
+import { metrics } from '../theme/spacing'
 
 const Chevron = ({ color }: { color: string }) => (
-    <Svg width="37" height="11" viewBox="0 0 37 11" fill="none">
+    <Svg
+        width="37"
```

Comment:
Is the value of the 37 in width and viewbox linked? I.e should they always be the same? If so worth pulling out into a const to make this clear.

Reply:
it's a bit magical at the moment, right now it's bound by height so the svg shrinks down and the width never comes into play